contracts/epochs/FlowEpoch.cdc

```diff
@@ -192,11 +197,15 @@ pub contract FlowEpoch {
         /// The number of collector clusters in each epoch
         pub(set) var numCollectorClusters: UInt16
 
-        init(numViewsInEpoch: UInt64, numViewsInStakingAuction: UInt64, numViewsInDKGPhase: UInt64, numCollectorClusters: UInt16) {
+        /// Tracks the percentage of FLOW total supply that is minted as rewards at the end of an epoch
```

Comment:
To me the comment implies that this value is the per-epoch percentage, but [here](https://github.com/onflow/flow-core-contracts/pull/132/files#diff-1da2c6b9c85ee2833c92c917dd4f57655a9f505ff07e2d8088977b6bcdb1969bR395) we divide by 52. Can make it clearer in the comment that this is the annual inflation rate